ajax/libs/vivid/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vivid",
+  "npmName": "vivid-icons",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js",
+        "*.json"
```

Comment:
Let's use `**/!(*.json|*.html)` to only include js files according to https://webkul.github.io/vivid/docs.html#installation